CHANGES.txt

```diff
@@ -1,5 +1,71 @@
-SciMath CHANGELOG
-=================
+SciMath Change Log
+==================
+
+Release 5.0.0
+-------------
+
+This is a major release that is focused around code clean-up and modernization.
+
+Much of the work was around removing deprecated and broken components, while
+updating code to remove usage of deprecated things from Python and ETS.  In
+particular, this release drops support for Python versions 3.5 and earlier.
+
+Additionally, the scimath.units.plugin sub-package has been removed: it was
+broken and removing it permitted a simplification of dependencies: we don't
+require Envisage and its dependencies in this release.
+
+Change summary since 4.2.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enhancements
+
+* move `scipy,interpolate` profiling script into its own module (#153)
+* remove unused `convert_quantity_old` (#139)
```

Comment:
Nit: if the contents of this file are intended to be reST (e.g., because we're intending to include it in the built docs) then we should be using double backticks everywhere.

OTOH, for easy pasting into the GitHub Release page, we want Markdown. :-(